recipes/packages.rb

```diff
@@ -118,35 +118,8 @@
                   'AIRFLOW_HOME' => node['airflow']['base_dir']})
     code <<-EOF
       set -e
-      #{node['conda']['base_dir']}/envs/airflow/bin/pip install --no-cache-dir apache-airflow["#{operator}"]==#{node['airflow']['version']}
+      #{node['conda']['base_dir']}/envs/airflow/bin/pip install apache-airflow["#{operator}"]==#{node['airflow']['version']} --constraint https://raw.githubusercontent.com/apache/airflow/1.10.10/requirements/requirements-python3.6.txt
     EOF
   end
 end
 
-# Install Airflow packages
-node['airflow']['packages'].each do |_key, value|
-  value.each do |val|
-    package_to_install = ''
-    version_to_install = ''
-    val.each do |k, v|
-      if k.to_s == 'name'
-        package_to_install = v
-      else
-        version_to_install = v
-      end
-    end
-    bash 'install_python__' + package_to_install do
-      umask "022"
-      user node['conda']['user']
-      group node['conda']['group']
-      code <<-EOF
-        set -e
-        #{node['conda']['base_dir']}/envs/airflow/bin/pip install --no-cache-dir \'#{package_to_install}#{version_to_install}\'
-      EOF
-    end
-    #python_package package_to_install.to_s do
-    #  action :install
-    #  version version_to_install.to_s
-    #end
-  end
-end
```

Comment:
Since we are not using the packages anymore, can you remove the dictionary from here (https://github.com/logicalclocks/airflow-chef/blob/d2cd4d3214777cd92142a9f7b8d143abb02367f5/attributes/packages.rb#L14) as well?

Reply:
fxd